docs/sources/guides/whats-new-in-v7-0.md

```diff
@@ -12,8 +12,160 @@ weight = -17
+
+Depending on the cardinality of the data and the time range MetricTank may return rolled up data (in our case, configured for hourly). This can be subtle as potentially only 1 or 2 graphs out of nine are rolled up.
+
+https://github.com/grafana/grafana/pull/22738
+
+## Breaking change - PhantomJS removed
+
+[PhantomJS](https://phantomjs.org/), have been used for rendering images of dashboards and panels and have been included with Grafana since Grafana v2.0. Since then we’ve had a lot of related bugs and security related issues, mainly due to the fact that PhantomJS have struggled with supporting modern web technologies. Throughout the years, maintaining PhantomJS support in Grafana has been a nightmare. Removing support for PhantomJS has been a high priority for the Grafana project and got stressed even more when the PhantomJS maintainer in March 2018 [announced](https://github.com/ariya/phantomjs/issues/15344) the end of the project.
+
+Since then we have been working towards removing PhantomJS. In October 2019, when Grafana v6.4 was released, we [announced](https://grafana.com/blog/2019/10/02/grafana-v6.4-released/#phantomjs-deprecation) the deprecation of PhantomJS. Grafana v7.0 removes all PhantomJS support which means that Grafana distribution no longer will include a built-in image renderer.
+
+As a replacement for PhantomJS we’ve developed the [Grafana Image Renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer) which is a plugin that runs on the backend and handles rendering panels and dashboards as PNG images using headless Chromium/Chrome. The [Grafana Image Renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer) can either be installed as a Grafana plugin running in its own process side-by-side with Grafana. or run as an external HTTP service, hosted using Docker or as a standalone application.
+
+Read more about [Image Rendering]({{< relref "../administration/image_rendering/" >}}) in the documentation for further instructions.
+
+## Rich History in Explore out of beta
+
+## Stackdriver data source supports Service Monitoring
+
+[Service monitoring](https://cloud.google.com/service-monitoring) in Google Cloud Platform (GCP) enables you to monitor based on Service Level Objectives (SLOs) for your GCP services. The new SLO query builder in the Stackdriver data source allows you to display SLO data in Grafana.
+
+https://grafana.com/docs/grafana/latest/features/datasources/stackdriver/#slo-service-level-objective-queries
+
+## Time zone support
+
+### Grafana Enterprise
+
+Grafana Enterprise focuses on solving problems for large companies and Grafana installations. And in Grafana 7.0 we are finally
+solving one of the most common problems of using Grafana at scale.
+
+That is problems like:
+
+- There are too many dashboards, how do I find the right one?
+- How to find popular dashboards
+- How to indentify dashboards that are not being used
+- Who created or last viewed this dashboard?
```

Comment:
this part was written super quick to get something before my sick daughter woke up :) needs further revisions :)

Reply:
We will take a stab at it